src/main/java/com/faforever/client/mod/ModManagerController.java

```diff
@@ -86,6 +97,22 @@ public void initialize() {
     loadActivatedMods();
 
     modListView.scrollTo(modListView.getSelectionModel().getSelectedItem());
+
+    modSearchTextField.textProperty().addListener(new ChangeListener<String>() {
+      @Override
+      public void changed(ObservableValue<? extends String> observable, String oldValue, String newValue) {
+        if (newValue.isEmpty()) {
+          modVersionFilteredList.setPredicate(null);
+        } else {
+          modVersionFilteredList.setPredicate(modVersion -> modVersion.getDisplayName().toLowerCase().contains(newValue.toLowerCase()));
```

Comment:
We would want the filter to search only the ui mods or sim mods based on which button was selected

Reply:
fixed